# Allow as many spans as Zaptel can take (defualt: 128).
# There is a static array of zt_pris allocated at this size.
# For that reason upstream only sort-of merged it: made it a
# compile-time option in 1.6 .

--- a/channels/chan_zap.c
+++ b/channels/chan_zap.c
@@ -190,7 +194,7 @@ static const char config[] = "zapata.con
 #define SIG_GR303FXOKS	(0x0100000 | ZT_SIG_FXOKS)
 #define SIG_GR303FXSKS	(0x0100000 | ZT_SIG_FXSKS)
 
-#define NUM_SPANS 		32
+#define NUM_SPANS 		ZT_MAX_SPANS
 #define NUM_DCHANS		4	/*!< No more than 4 d-channels */
 #define MAX_CHANNELS	672		/*!< No more than a DS3 per trunk group */
 
